checklist.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Remove this line from your PR.

Reply:
I'm sorry, but I don't understand which particular line